regression/goto-instrument-wmm-core/Makefile

```diff
@@ -21,7 +21,7 @@ tests.log: ../test.pl
 clean:
 	@for dir in *; do \
                 if [ -d "$$dir" ]; then \
-                        rm $$dir/*.txt $$dir/*.dot $$dir/*.gb $$dir/*.out; \
+                        rm -f $$dir/*.txt $$dir/*.dot $$dir/*.gb $$dir/*.out; \
```

Comment:
It may be worthwhile going straight for `$(RM)` (possibly everywhere) as this will be appropriately defined on each platform.